OpenRA.Game/Map/MapGrid.cs

```diff
@@ -107,6 +107,7 @@ public class MapGrid : IGlobalModData
 	{
 		public readonly MapGridType Type = MapGridType.Rectangular;
 		public readonly Size TileSize = new(24, 24);
+		public readonly int TileScale;
```

Comment:
This is partly my fault, but this is now exposed to be configurable from YAML, yet any value you input will be overriden by the default values in the constructor.
I guess it either needed to be a property (just elsewhere), or you remove the assignment in the constructor and define values in mod.yaml. Personally I prefer the latter.

Reply:
Ah yes. This was the reason, I didn't want it to be exposed to yaml

Reply:
reverted and moved